sdk/communication/communication-phone-numbers/src/phoneNumbersClient.ts

```diff
@@ -130,9 +136,8 @@ export class PhoneNumbersClient {
       options
     );
     try {
-      const { _response, ...results } = await this.client.getByNumber(phoneNumber, updatedOptions);
-      return results;
-    } catch (e) {
+      return await this.client.phoneNumbers.getByNumber(phoneNumber, updatedOptions);
```

Comment:
Correct if I'm wrong but by waiting this function call aren't you break the return type? or is this function returning a promise after you wait for it?

Reply:
I don't think that this changes the return type; even if the call is awaited, the return is still wrapped around a Promise. Also, the previous implementation also awaited the call; the only difference is that it assigned the return value to a variable first in order to filter out the `_response` property from the results (which is no longer necessary since that property is no longer used in Core V2).